### PR TITLE
Remove Profiles feature from WebApp

### PR DESCRIPTION
**Pull request by borkopetrovicc**
## Description

This PR solves # .

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx [hunk 1]
```diff
@@ -5,20 +5,17 @@ import { useTranslation } from 'react-i18next';
...
 import useNotifications from '#src/hooks/useNotifications';
 
 const RoutesContainer = () => {
   const { i18n } = useTranslation();
   const location = useLocation();
-  const userModal = useQueryParam('u');
 
   const { profile, selectingProfileAvatar } = useProfileStore();
-  const { shouldManageProfiles } = useProfiles();
+  useProfiles();
```
Comment (AntonLantukh) [inline platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx]: Can we fully remove them from the repo?

Comment (darkoatanasovski) [reply platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx]: and the line above should be removed as well :)

File: platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx [hunk 2]
```diff
@@ -41,11 +38,6 @@ const RoutesContainer = () => {
     return <LoadingOverlay profileImageUrl={selectingProfileAvatar || profile?.avatar_url} />;
```
Comment (darkoatanasovski) [inline platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx]: also remove this

File: platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx (hunk 1 above)
Comment (AntonLantukh) [reply platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx]: @borkopetrovicc I think we also have a service and a controller for profiles.

Comment (darkoatanasovski) [reply platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx]: @AntonLantukh I know, but as we discussed, we will just remove the visual flow for now. In Q3, we will make an additional effort for complete removal

Comment (AntonLantukh) [reply platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx]: @darkoatanasovski I think visual part is still there:

Screenshot 2024-06-06 at 13 33 48

Comment (borkopetrovicc) [reply platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx]: a2246a4929c146792113f23c8a52749973cd267f.  @AntonLantukh

**Comment by AntonLantukh** [comment]
@borkopetrovicc I still see some profile parts:

Screenshot 2024-06-06 at 22 12 29
Screenshot 2024-06-06 at 22 12 39

**Comment by borkopetrovicc** [comment]
> @borkopetrovicc I still see some profile parts:
> 
> Screenshot 2024-06-06 at 22 12 29 Screenshot 2024-06-06 at 22 12 39

ce7e78b2bf4060b591255416736cb4fd8b4bc2b3

**Comment by AntonLantukh** [comment]
Please also remove Profile paths:

```
        <Route path={PATH_USER_PROFILES} element={<Profiles />} />
        <Route path={PATH_USER_PROFILES_CREATE} element={<CreateProfile />} />
        <Route path={PATH_USER_PROFILES_EDIT} element={<Profiles editMode />} />
        <Route path={PATH_USER_PROFILES_EDIT_PROFILE} element={<EditProfile />} />

```

**Review by AntonLantukh — APPROVED** [review]

**Comment by ChristiaanScheermeijer** [comment]
This commit won't show in the changelog :')

**Pull request merged**